environment.yml

```
@@ -0,0 +1,18 @@
+name: UKBiobankGWAS 
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - r-base=4.2
+  - r-essentials
+  - python=2.7
```

Python 2.7 is no longer supported by Conda, and many packages have been removed. Cannot create this environment with 'conda env create -f environment.yml'

---

that's weird, cause I was able to make this and store the env on bc4.  but i didn't use it, so maybe it silently failed.